developer-docs-site/docs/nodes/validator-node/operator/delegation-pool-operations.md

```diff
@@ -0,0 +1,193 @@
+
+  ```bash
+  aptos move run --profile ‘delegator’ \
+  --function-id 0x1::delegation_pool::withdraw \
+  --args address:’pool_address’ u64:’amount’
+  ```
+
+## Pool owner operations
+ 
+Delegation pool owners have access to specific methods designed for modifying the `operator` and `voter` roles of the delegation pool. Use the following Aptos CLI commands and include the relevant addresses:
+
+  ```bash
+  aptos move run --profile ‘delegation_pool_owner’ \
+  --function-id 0x1::delegation_pool::set_operator \
+  --args address:’new_operator_address’
+  ```
+  
+  ```bash
+ aptos move run --profile ‘delegation_pool_owner’ \
+ --function-id 0x1::delegation_pool::set_delegated_voter \
+ --args address:’new_delegated_voter_address’
+  ```
+  
+## Check delegation pool information
+
+Until the delegation pool has received 1M APT and the validator has been added to the set of active validators, there will be no rewards to track during each cycle. In order to obtain information about a delegation pool, use the Aptos [View functon](../../../guides/aptos-apis.md#reading-state-with-the-view-function).
+
+1. `get_owned_pool_address(owner: address): address` -  Returns the address of the delegation pool belonging to the owner, or produces an error if there is no delegation pool associated with the owner.
+
+2. `delegation_pool_exists(addr: address): bool` - Returns if a delegation pool exists at the provided address `addr`.
+
+3. `operator_commission_percentage(pool_address: address): u64` - Returns the operator commission percentage set on the delegation pool at initialization.
+
+4. `get_stake(pool_address: address, delegator_address: address): (u64, u64, u64)` - Returns total stake owned by `delegator_address` within delegation pool `pool_address` in each of its individual states: (`active`,`inactive`,`pending_inactive`).
+
+5. `get_delegation_pool_stake(pool_address: address): (u64, u64, u64, u64)` - Returns the stake amounts on `pool_address` in the different states:      (`active`,`inactive`,`pending_active`,`pending_inactive`).
+
+6. `shareholders_count_active_pool(pool_address: address): u64` - Returns the number of delegators owning an active stake within `pool_address`. 
+
+7. `get_pending_withdrawal(pool_address: address, delegator_address: address): (bool, u64)` - Returns if the specified delegator possesses any withdrawable stake. However, if the delegator has recently initiated a request to release some of their stake and the stake pool's lockup cycle has not ended yet, then their funds may not yet be available for withdrawal.
+
+8. `can_withdraw_pending_inactive(pool_address: address): bool` - Returns whether `pending_inactive` stake can be directly withdrawn from the delegation pool, implicitly its stake pool, in the special case the validator had gone inactive before its lockup expired.
+
+
+In the [Aptos TypeScript SDK](../../../sdks/ts-sdk/index.md), a View function request would resemble:
```

Comment:
Alternatively, you can call the above functions as view functions using aptos move view